javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,141 @@
+//Voice assistant
+let personName = "";
+let todoList = [];
+
+function getReply(command) {
+	var options = {
+		// weekday: "long",
+		year: "numeric",
+		month: "long",
+		//day: "numeric",
+	};
+	var today = new Date();
```

Comment:
general rule: always declare variables with const.
If you think the value of the variable can change, use let.
Use of var is not a preferred method anymore after ES6